Lib/test/test_importlib/resources/util.py

```diff
@@ -5,8 +5,7 @@
 import types
 from pathlib import Path, PurePath
 
-from .. import data01
-from .. import zipdata01
```

Comment:
This file is a library file.  It is not meant to be run as a script.  So the relative imports are fine and there's no need to change anything here.